.github/workflows/integration.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     env:
       GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       LOCKBOOK_CI: 1
```

Comment:
[github](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) has an env var for CI already we should just use that and avoid thinking and dealing with this.